src/components/Goban.js

```diff
@@ -262,9 +262,34 @@ export default class Goban extends Component {
         board.height
       )
 
+    // Calculate coordinates
+
+    let relativeCoord = (x, size) => {
```

Comment:
For fewer top-level variables and cleaner code, I suggest you move this function into the `else if (coordinatesType === 'relative')` branch.